app/controllers/events_controller.rb

```diff
@@ -17,10 +17,9 @@ def show
   end
 
   def index
-    respond_to do |format|
-      format.html {  @events = Event.upcoming(10) }
-      format.json {  @events = Event.where(start_date: params[:start]..params[:end]) }
-    end
+    @events = Event.upcoming(10)
+    @markers = BuildMarkersWithInfoWindow
+      .with(Event.build_by_coordinates(@events), self) 
```

Comment:
I notice we lose the json format here - fair enough - it's not covered by any tests - not a feature requested from the client ...

Reply:
I updated the changes per your request. With the exception of this one. Judging by your comments, I wasn't sure it should be updated. Please advise. Thanks.

Reply:
actually, I just submitted the fix for this as well.